src/cli/dsc/schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Strongly recommend using 2020-12 for your schema, which supports much cleaner expression of data types, composition, and validation.